table/block_based_table_factory.cc

```diff
@@ -191,13 +197,28 @@ std::string BlockBasedTableFactory::GetPrintableTableOptions() const {
   snprintf(buffer, kBufferSize, "  index_block_restart_interval: %d\n",
            table_options_.index_block_restart_interval);
   ret.append(buffer);
+  snprintf(buffer, kBufferSize, "  metadata_block_size: %" PRIu64 "\n",
```

Comment:
I believe you can use ROCKSDB_PRIszt here.

Reply:
metadata_block_size is a uint64_t
https://github.com/facebook/rocksdb/blob/master/include/rocksdb/table.h#L156

Reply:
Oh ok. I thought it would have the same type as block_size, which is printed a few lines above.